test/unit-test/dawn/Optimizer/CMakeLists.txt

```diff
@@ -17,11 +17,14 @@ dawn_add_unittest_impl(
   SOURCES TestColoringAlgorithm.cpp
           TestExtent.cpp  
           TestEnvironment.h
+          TestField.cpp
           TestGraph.cpp
           TestIsDAGAlgorithm.cpp          
           TestPartitionAlgorithm.cpp
           TestMain.cpp
           TestInterval.cpp
+          TestIntervalAlgorithms.cpp
```

Comment:
did not find this file anywhere

Reply:
thanks, I forgot to add it. Now you have one more file to review ;)